CMSgov/mint-app
# [NOREF] bumping the limit for a model plan's name

# NOREF


## Changes and Description
This addresses https://cmsgov.slack.com/archives/C04B10ZN6A2/p1714587884856889?thread_ts=1714586619.597419&cid=C04B10ZN6A2

- Bumping up the model plan name character count limit from 50 to 200 




## How to test this change

1. Create a new model plan
2. Name it with an extremely long name
3. Save
4. Go back and try to rename the model name
5. Verify that long name save works



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
